internal/core/common_config/main.go

```diff
@@ -99,3 +148,74 @@ func translateInterruptToCancel(ctx context.Context, wg *sync.WaitGroup, cancel
 		}
 	}()
 }
+
+func loadYaml(lc logger.LoggingClient, yamlFile string, configClient configuration.Client) error {
+	// push not done flag to configClient
+	err := configClient.PutConfigurationValue(commonConfigDone, []byte("false"))
+	if err != nil {
+		return fmt.Errorf("failed to push %s on startup: %s", commonConfigDone, err.Error())
+	}
+	lc.Infof("reading %s", yamlFile)
+	contents, err := os.ReadFile(yamlFile)
```

Comment:
prefer to normalize this input yamlFile path into the absolute path form before opens it

Reply:
No we don't do that unless the user uses the in the flags. This is consistent with how bootstrap does it for all the other services.

Reply:
if the end user specifies the input yamlfile location at free will: it could be something like `../../res/common-configu-file/`  

if that path is kind of built-in then probably it is ok.

Reply:
We must support relative paths.

Reply:
yes, absolutely. what i was suggesting is to show the absolute path in logging and before opens it. as it could just find the error in the path via 
```go
  yamlFileAbs, err := filepath.Abs(yamlFile)
```
before the os opens it.  

I am ok not a big deal issue and just suggestion.

Reply:
> if that path is kind of built-in then probably it is ok.

Yes, common function in bootstrap.